rules/ml/persistence_ml_rare_process_by_host_linux.toml

```diff
@@ -26,11 +26,54 @@ machine_learning_job_id = ["v3_rare_process_by_host_linux"]
 name = "Unusual Process For a Linux Host"
 note = """## Triage and analysis
 
-### Investigating an Unusual Linux Process
-Detection alerts from this rule indicate the presence of a Linux process that is rare and unusual for the host it ran on. Here are some possible avenues of investigation:
-- Consider the user as identified by the username field. Is this program part of an expected workflow for the user who ran this program on this host?
-- Examine the history of execution. If this process only manifested recently, it might be part of a new software package. If it has a consistent cadence (for example if it runs monthly or quarterly), it might be part of a monthly or quarterly business process.
-- Examine the process arguments, title and working directory. These may provide indications as to the source of the program or the nature of the tasks it is performing."""
+###  Investigating Unusual Process For a Linux Host
+
+Searching for abnormal Linux processes is a good methodology to find potentially malicious activity within a network.
+Understanding what is commonly run within an environment and developing baselines for legitimate activity can help
+uncover potential malware and suspicious behaviors.
+
+This rule uses a machine learning job to detect a Linux process that is rare and unusual for an individual Linux host in your fleet.
+#### Possible investigation steps
```

Comment:
Quick question about this sentence - in this instance, is this the Elastic Fleet or just a general group (i.e., a fleet of hosts). If it's the former, I'd capitalize the "f" in fleet. If it's the latter, it might be worth choosing a different term since "Fleet" is used in a very specific context within Elastic products. For example:

```suggestion
This rule uses a machine learning job to detect a Linux process that is rare and unusual for an individual Linux host within your group of hosts.

#### Possible investigation steps
```

Reply:
Replaced it with `environment`.